.github/scripts/display_ec2_information.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function get_ec2_metadata() {
+    category=$1
+    curl -fsSL "http://169.254.169.254/latest/meta-data/${category}"
```

Comment:
Can you add a comment about where this is hosted?

Reply:
Done